nomos-da/network/subnetworks-assignations/src/versions/v2.rs

```diff
@@ -63,7 +65,7 @@ impl FillWithOriginalReplication {
 }
 
 impl MembershipHandler for FillWithOriginalReplication {
-    type NetworkId = u16;
+    type NetworkId = u32;
```

Comment:
There's `SubnetworkId` type in `nomos-da-network-core`, maybe we could use it in all network related places?

Reply:
Yes. I changed it as it was a miss aligned type. It could probably be changed in a separated PR though.